### ISSUE TITLE
Errors in WindowsPlatformSpecifics when using EOS_DISABLE define

### ISSUE THREAD

**Issue by jollypunch** [PlayEveryWare/eos_plugin_for_unity#687]
Errors when using EOS_DISABLE define, similar to https://github.com/PlayEveryWare/eos_plugin_for_unity/issues/653

```
Library\PackageCache\com.playeveryware.eos@74682501a1\Runtime\Windows\Core\WindowsPlatformSpecifics.cs(39,7): error CS0246: The type or namespace name 'Epic' could not be found (are you missing a using directive or an assembly reference?)

Library\PackageCache\com.playeveryware.eos@74682501a1\Runtime\Windows\Core\WindowsPlatformSpecifics.cs(54,16): error CS0246: The type or namespace name 'WindowsOptions' could not be found (are you missing a using directive or an assembly reference?)

Library\PackageCache\com.playeveryware.eos@74682501a1\Runtime\Windows\Core\WindowsPlatformSpecifics.cs(59,16): error CS0246: The type or namespace name 'InitializeOptions' could not be found (are you missing a using directive or an assembly reference?)

Library\PackageCache\com.playeveryware.eos@74682501a1\Runtime\Windows\Core\WindowsPlatformSpecifics.cs(63,45): error CS0246: The type or namespace name 'PlatformSpecifics<>' could not be found (are you missing a using directive or an assembly reference?)

Library\PackageCache\com.playeveryware.eos@74682501a1\Runtime\Windows\Core\WindowsPlatformSpecifics.cs(88,30): error CS0115: 'WindowsPlatformSpecifics.LoadDelegatesWithEOSBindingAPI()': no suitable method found to override

Library\PackageCache\com.playeveryware.eos@74682501a1\Runtime\Windows\Core\WindowsPlatformSpecifics.cs(105,30): error CS0115: 'WindowsPlatformSpecifics.ConfigureSystemInitOptions(ref EOSInitializeOptions, EOSConfig)': no suitable method found to override

Library\PackageCache\com.playeveryware.eos@74682501a1\Runtime\Windows\Core\WindowsPlatformSpecifics.cs(118,30): error CS0115: 'WindowsPlatformSpecifics.ConfigureSystemPlatformCreateOptions(ref EOSCreateOptions)': no suitable method found to override
```

**Comment by arthur740212** [comment]
Thanks for the feedback.
We have tracked this issue and it should be fixed in the next release.

**Comment by paulhazen** [comment]
@jollypunch 

This issue is resolved in our upcoming `3.2.0` release, which should be out early next week. In the meantime, you can find the changes that fix the issue (as well as other improvements) in the `release-3.2.0` branch. Accordingly, I'm going to label this issue as "pending," and close it. Feel free to re-open if after migrating to the newer release you are still experiencing the same problem.
